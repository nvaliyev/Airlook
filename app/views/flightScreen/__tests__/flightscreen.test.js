import 'react-native';
import React from 'react';
import { act, create } from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';
import FlightScreen from '..';
describe('Flightscreen', () => {
    test('Flightscreen snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<FlightScreen/>);
        expect(snapshot).toMatchSnapshot();
    });
});


