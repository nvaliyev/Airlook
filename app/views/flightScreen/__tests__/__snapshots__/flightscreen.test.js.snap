// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flightscreen Flightscreen snapshot 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "alignItems": "center",
      "backgroundColor": "#f7f9fc",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
     FlightScreen 
  </Text>
</View>
`;
